sematic/api/endpoints/runs.py

```diff
@@ -238,6 +337,8 @@ def save_graph_endpoint(user: Optional[User]):
     graph = flask.request.json["graph"]
 
     runs = [Run.from_json_encodable(run) for run in graph["runs"]]
+    for run in runs:
+        logger.info("Graph update, run %s is in state %s", run.id, run.future_state)
```

Comment:
We will be thankful later (when debugging hairy issues) to be able to see when run states get moved!